docs/dashboard/orders.md

```diff
@@ -136,9 +138,21 @@ When order confirmation is enabled, order needs to be confirmed by staff in orde
 
 1. Select an unconfirmed order from order list to open the order details
 
-2. Click _confirm&nbsp;order_ button on the bottom
+2. There is a possiblity to edit the order at this point. However there are a few things to be aware when doing so:
+
+- When order address gets changed, existing allocations remains untouched.
+- New allocations are created whenever staff increase order line amount or add a new order line.
+- Allocations will only get altered for variants with active inventory tracking set.
+- When order line is deleted, all existing allocations of that order line are deleted with it.
+- If staff tries to create / increase amount on an order line with greater amount of variant that is available, insufficient stock error will be displayed.
+- When manipulating order lines after order address gets changed, there might be a case that the items cannot be allocated for new address.
+
+3. Click _confirm&nbsp;order_ button on the bottom
 
 After operation is finished, order status should be _unfulfilled_.
+Keep in mind that staff might change the order value before it's confirmed by manipulating the order lines. If such case happens, then after the confirmation, charged amount can be:
+- higher than order total - staff should perform miscellaneous refund for the amount that was overpaid.
+- lower than order total - staff should create a separate payment in order to clear the ballance.
```

Comment:
```suggestion
- lower than order total - staff should create a separate payment in order to clear the balance.
```

Reply:
"staff should create a separate payment" - if I understand correctly, it suggests that Saleor supports creating multiple payments for an order, but this is not possible. Staff users cannot create separate payments, if there is already a partial payment. I think in this case we want staff users to decide with the customer how to handle outstanding balance.